* because <linux/input.h> is compiled in host too,
we had to remove <asm/types.h> from input.h
so include it now.

see more details at linux package patch.

diff -Naur lirc-0.8.0.orig/daemons/hw_devinput.c lirc-0.8.0/daemons/hw_devinput.c
--- lirc-0.8.0.orig/daemons/hw_devinput.c	2006-03-20 21:24:53.000000000 +0100
+++ lirc-0.8.0/daemons/hw_devinput.c	2006-03-20 22:09:10.000000000 +0100
@@ -32,6 +32,7 @@
 #include <dirent.h>
 #include <fnmatch.h>
 
+#include <asm/types.h>
 #include <linux/input.h>
 
 #ifndef EV_SYN
